src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
Well, `node` and `nodes` aren't technically public API are they? We don't document them as far as I can see. I agree that using a function would be an improvement, so maybe we can make that change for the next breaking release and then add `node()` and `nodes()` to the API documentation?